package.json

```diff
@@ -21,11 +22,14 @@
   },
   "devDependencies": {
     "bluebird": "^2.9.12",
+    "chai": "^2.2.0",
```

Comment:
Please keep the test code consistent and use the `should` module in the memory leak tests too.